### ISSUE TITLE
RS90 - eep files appear in file browser

### ISSUE THREAD

**Issue by Cralex** [gameblabla/pokemini#1]
Because save files appear alongside ROMs, the file browser shows both. For example, playing Pokémon Tetris.zip (displayed as Pokémon Tetris in gmenu) generates Pokémon Tetris.zip.eep, which appears as Pokémon.zip in the ROM browser.

**Comment by gameblabla** [comment]
You mean in the internal ROM browser or when using Gmenu2x ?

**Comment by Cralex** [comment]
Using Gmenu2x. The emulator’s built-in browser properly ignores save files. I’m ignorant about if the RS90’s port of Gmenu supports filtering by file extensions (super easy fix) or if the only option would be to redirect the save files elsewhere.

Anyway, trying to open a save file by accident causes Pokemini to open without a cart loaded.

https://www.dropbox.com/s/6pfqbe1p9v1jifb/Photo%20May%2005%2C%2011%2047%2033%20AM.jpg?dl=1

**Comment by gameblabla** [comment]
Fixed as of https://github.com/gameblabla/pokemini/commit/5d0064b52e3d5e2f2abdbcbf3d4edbdfd47108c1. The OPK file also got updated on my website so download it again. Open again if it still happens.

**Comment by Cralex** [comment]
Unfortunately, gmenu doesn’t seem to be acknowledging your settings. I installed the updated opk, rebooted the unit, then tried playing two more games for the first time. Same thing happened. 

https://www.dropbox.com/s/scqhu2nmgh8x2ji/Photo%20May%2005%2C%201%2002%2019%20PM.jpg?dl=0

Also, the save config worked for regular settings (palette, for example) but the scaling mode did not appear to save. The dialog said that the config saved, but fullscreen was off when I restarted the emulator.

**Comment by gameblabla** [comment]
Oh yeah, it doesn't save the scaling setting. And i'm surprised that it would ignore the selectorfilter setting unlike GmenuNx... If so then i'm afraid it won't be an easy fix.

**Comment by Cralex** [comment]
Understood, just as long as you’re aware of it. :) Maybe shared eep could be turned on by default to mitigate the problem in the meantime? I just played Psychic Seeds with the shared eep setting on and I didn’t notice another file appearing... I’ll need to get to a computer to see where the shared eep is going, but it does seem to work. (Either that, or maybe a Gmenu update will come along to fix this.)

**Comment by gameblabla** [comment]
Yeah i guess we could do that instead though it's not a true fix. (i was thinking of moving those .epm files to a . home directory but that does not to be easily feasible here. I'm not too familiar with this codebase)

**Comment by Cralex** [comment]
I manually edited my pokemini.cfg file and it seems to have fixed the problem.

eepromfile=/media/data/local/home/.pokemini/shared.eep
eepromshare=yes

This’ll generate a shared eeprom right next to the emulator’s own config file on internal storage. Looks like eepromfile automatically sets itself to the name and location of the first game you run, so simply turning shared eep on in the menu will just make it recreate the same eep file over and over if you try to clean them out of the ROM directory. Maybe make these changes in the default config and remove the shared eep option from the in-game menu? (At least until you actually want to dig through the codebase to move the individual eep files. Personally, shared eep is good enough for me.)

**Comment by gameblabla** [comment]
That's a hack though and i would like to fix this the proper way, which is to generate said eep files in the home directory.
